x/auth/tx/service_test.go

```diff
@@ -184,6 +184,16 @@ func TestEventRegex(t *testing.T) {
 			event: "tx.signature='foobar/baz123=='",
 			match: true,
 		},
+		{
```

Comment:
[Non blocking][Question]
Does it make sense to add tests also for the other operators supported by core?

https://github.com/celestiaorg/celestia-core/blob/3e636bb02f48cc02b11f4aea3dd1494ef8e62c91/libs/pubsub/query/query.go#L65-L84

Even if they're caught by the regex, so that upcoming changes don't end up breaking the API again

Reply:
Wow great question! I think we should do this in a follow-up b/c I don't want to dismiss approvals. TBH I'd be a little surprised if "EXISTS" and "CONTAINS" work b/c the current regex doesn't special case for those.